CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unreleased]: https://github.com/atomist-rugs/rug-editors/compare/0.11.0...HEAD
 
+- Added `ConvertExistingProjectToGenerator` editor.
```

Comment:
All changes should now be in, and assuming it's all good to go then this will be a squash and merge if you don't mind @ddgenome ?